daily_post.py

```diff
@@ -5,6 +5,7 @@
 import praw
 import requests
 import time
+import daily_markov
```

Comment:
Sorry, again OCD, alpha order on imports

Reply:
I swear one of these days I'll remember.